front_end/src/app/(main)/questions/[id]/components/forecast_maker/forecast_maker_conditional/forecast_maker_conditional_continuous.tsx

```diff
@@ -61,10 +65,14 @@ const ForecastMakerConditionalContinuous: FC<Props> = ({
   const questionYesId = question_yes.id;
   const questionNoId = question_no.id;
 
-  const prevYesForecastValue = extractPrevNumericForecastValue(prevYesForecast);
-  const prevNoForecastValue = extractPrevNumericForecastValue(prevNoForecast);
+  const latestYes = question_yes.my_forecasts?.latest;
+  const latestNo = question_no.my_forecasts?.latest;
+  const prevYesForecastValue =
+    latestYes && !latestYes.end_time ? latestYes.slider_values : null;
+  const prevNoForecastValue =
+    latestNo && !latestNo.end_time ? latestNo.slider_values : null;
```

Comment:
Similar notes about `extractPrevNumericForecastValue` and `prevYesForecast`, `prevNoForecast` props